synapse/models/person.py

```diff
@@ -409,6 +418,25 @@ def getModelDefs(self):
                         'doc': 'A crypto currency address associated with the contact.'
                     }),
                 )),
+                ('ps:vitals', {}, (
+                    ('asof', ('time', {}), {
+                        'doc': 'The time the vitals were gathered or computed.'}),
+                    ('contact', ('ps:contact', {}), {
+                        'doc': 'The contact that the vitals are about.'}),
+                    ('person', ('ps:person', {}), {
+                        'doc': 'The person that the vitals are about.'}),
+                    ('height', ('geo:dist', {}), {
+                        'doc': 'The height of the person or contact.'}),
+                    ('weight', ('mass', {}), {
+                        'doc': 'The weight of the person or contact.'}),
+                    ('currency', ('econ:currency', {}), {
+                        'doc': 'The currency that the price values are recorded using.'}),
+                    ('net:worth', ('econ:price', {}), {
```

Comment:
For future-proofing, do we want to further "sub-categorize" ps:vitals props - e.g., "economic" vitals vs "bio" vitals vs ?? For example:
  :econ:currency
  :econ:net:worth
  :econ:annual:income
  :bio:blood:pressure
  :bio:blood:hemoglobin
  :bio:blood:hdl
  :bio:blood:ldl
  ...etc.?

Reply:
Sure, i'll add namespacing, thanks!